packages/gestalt/src/TextArea.jsdom.test.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import TextArea from './TextArea.js';
+
+describe('TextArea', () => {
+  it('TextArea with errorMessage prop change', () => {
+    const { getByText, rerender } = render(
+      <TextArea
+        id="test"
+        onChange={jest.fn()}
+        onFocus={jest.fn()}
+        onBlur={jest.fn()}
+      />
+    );
+    expect(() => {
+      getByText('Error message');
+    }).toThrow('Unable to find an element with the text: Error message');
```

Comment:
Is catching an exception really the right pattern for this? Without reading the docs, it feels like there should be a selector that just returns something falsy on no results.

Reply:
That's the way React Testing Library does it - haven't found a better way yet